doc/content/getting-started/migrating/major-changes.md

```diff
@@ -23,6 +23,18 @@ Furthermore, {{% tts %}} brings full support for all LoRaWAN versions, as well a
 
 Devices imported from {{% ttnv2 %}} are configured with an `Rx1Delay` of 1 second, by default. In {{% tts %}} we recommend using an `Rx1Delay` of 5 seconds to accommodate for high latency backhauls and/or [peering with Packet Broker]({{< ref "/reference/packet-broker" >}}). See the [MAC settings guide]({{< ref "/devices/mac-settings" >}}) for more information and instructions about configuring `Rx1Delay`.
 
+## MAC Commands
+
+{{% tts %}} expects that all end devices comply to the LoRaWAN spec, which means that the end devices should properly respond to all MAC commands requests issued by the Network Server. If a device fails to forward a response to MAC Command in a timely manner, there may be disruptions to the device uplink or downlink traffic. As mentioned in the LoRaWAN spec, the Network Server of {{% tts %}} will always prioritize MAC commands over device traffic.
```

Comment:
```suggestion
{{% tts %}} expects that all end devices comply with the LoRaWAN specification by default, which means that the end devices should respond to Network Server MAC command requests accordingly. If a device fails to answer a MAC Command in a timely manner, there may be disruptions to the device uplink or downlink traffic. As mentioned in the LoRaWAN specification, the Network Server of {{% tts %}} will always prioritize MAC commands over application payloads on downlink.
Note, that in case a device is not fully compliant with the specification - it can still work on {{% tts %}}, but it may require custom MAC settings configuration.
```